io.github.HotShots.yml

```diff
@@ -0,0 +1,44 @@
+app-id: io.github.HotShots
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+
+command: hotshots
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+  - --filesystem=xdg-pictures
+  - --share=network
+  
+modules:
+- name: HotShots
+  sources:
+  - type: git
+    url: https://github.com/obiwankennedy/HotShots
+
+  - type: file
+    path: io.github.HotShots.desktop
+  
+  - type: file
+    path: io.github.HotShots.metainfo.xml
+
+
+  buildsystem: simple
+  build-commands:
+   - qmake -recursive  HotShots.pro
+   - sed 's/\/usr\/local/\/app/g' -i Makefile.Release
+   - sed 's/\/usr\/local/\/app/g' -i Makefile.Debug
+   - make install
+   - mkdir -p ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps
+   - install -Dm644 /app/share/pixmaps/hotshots.png 
+                    /app/share/icons/hicolor/128x128/apps/io.github.HotShots.png
+   - install -Dm644 io.github.HotShots.desktop /app/share/applications/
+   
+   - mkdir -p /app/share/metainfo
+   - install -Dm644 io.github.HotShots.metainfo.xml /app/share/metainfo
+
+#  build-options:
+#    build-args:
+#    - "--share=network"
```

Comment:
Are those needed?

Reply:
Don't know, hence commented, thought network is required for git